### ISSUE TITLE
While running bash build.rb on local getting error for precompile

### ISSUE THREAD

**Issue by vishalzambre** [spree/spree#7094]
Here is the log this error is getting for each engine dummy app

```
Setup Spree defaults and creating test application...
Resolving dependencies....
The Gemfile's dependencies are satisfied
Generating dummy Rails application...
Setting up dummy database...
/home/vishal/.rvm/gems/ruby-2.2.3/gems/activerecord-4.2.5.1/lib/active_record/associations.rb:1694: warning: already initialized constant Spree::ShippingMethod::HABTM_Zones
/home/vishal/.rvm/gems/ruby-2.2.3/gems/activerecord-4.2.5.1/lib/active_record/associations.rb:1694: warning: previous definition of HABTM_Zones was here
Precompiling assets...
I, [2016-02-19T17:59:10.201752 #2171]  INFO -- : Writing /home/vishal/workspace/spree/api/spec/dummy/public/assets/spree/backend/orders/edit_form-dc98920d0593eedcd29c7f3baf235a71f06ff180c61947d6aa3badeba7855469.js
I, [2016-02-19T17:59:10.202461 #2171]  INFO -- : Writing /home/vishal/workspace/spree/api/spec/dummy/public/assets/spree/backend/orders/edit_form-dc98920d0593eedcd29c7f3baf235a71f06ff180c61947d6aa3badeba7855469.js.gz
I, [2016-02-19T17:59:10.208641 #2171]  INFO -- : Writing /home/vishal/workspace/spree/api/spec/dummy/public/assets/spree/backend/address_states-b1e1f0afd855bedef5faecce0a6510759c1f4d12b7ea1093104cafb885ac45b2.js
I, [2016-02-19T17:59:10.208800 #2171]  INFO -- : Writing /home/vishal/workspace/spree/api/spec/dummy/public/assets/spree/backend/address_states-b1e1f0afd855bedef5faecce0a6510759c1f4d12b7ea1093104cafb885ac45b2.js.gz
I, [2016-02-19T17:59:10.236361 #2171]  INFO -- : Writing /home/vishal/workspace/spree/api/spec/dummy/public/assets/select2-d6b5d8d83dbc18fb8d77c8761d331cd9e5123c9684950bab0406e98a24ac5ae8.png
I, [2016-02-19T17:59:10.241840 #2171]  INFO -- : Writing /home/vishal/workspace/spree/api/spec/dummy/public/assets/select2-spinner-f6ecff617ec2ba7f559e6f535cad9b70a3f91120737535dab4d4548a6c83576c.gif
I, [2016-02-19T17:59:10.264893 #2171]  INFO -- : Writing /home/vishal/workspace/spree/api/spec/dummy/public/assets/select2x2-6fe28d687dc0ed4d96016238c608ba1e7198c9c9accfa0b360b78018b9fb9bc2.png
I, [2016-02-19T17:59:10.270176 #2171]  INFO -- : Writing /home/vishal/workspace/spree/api/spec/dummy/public/assets/application-e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855.css
I, [2016-02-19T17:59:10.270308 #2171]  INFO -- : Writing /home/vishal/workspace/spree/api/spec/dummy/public/assets/application-e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855.css.gz
I, [2016-02-19T17:59:12.413148 #2171]  INFO -- : Writing /home/vishal/workspace/spree/api/spec/dummy/public/assets/spree/backend/all-41403aea06a49f2388d1cdd724522c4613af02cd7960343ba072ae9fb29aec20.js
I, [2016-02-19T17:59:12.413418 #2171]  INFO -- : Writing /home/vishal/workspace/spree/api/spec/dummy/public/assets/spree/backend/all-41403aea06a49f2388d1cdd724522c4613af02cd7960343ba072ae9fb29aec20.js.gz
I, [2016-02-19T17:59:12.962920 #2171]  INFO -- : Writing /home/vishal/workspace/spree/api/spec/dummy/public/assets/spree/frontend/all-61f87f2478173f4ed1039e66150aeadf9d5d2137857459e524d7cfcde41fdfb1.js
I, [2016-02-19T17:59:12.963194 #2171]  INFO -- : Writing /home/vishal/workspace/spree/api/spec/dummy/public/assets/spree/frontend/all-61f87f2478173f4ed1039e66150aeadf9d5d2137857459e524d7cfcde41fdfb1.js.gz
rake aborted!
Sprockets::FileNotFound: couldn't find file 'spree/backend/spree_api' with type 'text/css'
/home/vishal/workspace/spree/api/spec/dummy/vendor/assets/stylesheets/spree/backend/all.css:8
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/resolve.rb:64:in `resolve!'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/directive_processor.rb:399:in `resolve'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/directive_processor.rb:207:in `process_require_directive'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/directive_processor.rb:180:in `block in process_directives'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/directive_processor.rb:178:in `each'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/directive_processor.rb:178:in `process_directives'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/directive_processor.rb:83:in `_call'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/directive_processor.rb:68:in `call'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/processor_utils.rb:75:in `call_processor'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/processor_utils.rb:57:in `block in call_processors'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/processor_utils.rb:56:in `reverse_each'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/processor_utils.rb:56:in `call_processors'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/loader.rb:134:in `load_from_unloaded'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/loader.rb:60:in `block in load'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/loader.rb:318:in `fetch_asset_from_dependency_cache'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/loader.rb:44:in `load'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/cached_environment.rb:20:in `block in initialize'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/cached_environment.rb:47:in `yield'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/cached_environment.rb:47:in `load'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/bundle.rb:23:in `block in call'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/utils.rb:183:in `dfs'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/bundle.rb:24:in `call'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/processor_utils.rb:75:in `call_processor'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/processor_utils.rb:57:in `block in call_processors'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/processor_utils.rb:56:in `reverse_each'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/processor_utils.rb:56:in `call_processors'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/loader.rb:134:in `load_from_unloaded'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/loader.rb:60:in `block in load'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/loader.rb:318:in `fetch_asset_from_dependency_cache'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/loader.rb:44:in `load'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/cached_environment.rb:20:in `block in initialize'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/cached_environment.rb:47:in `yield'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/cached_environment.rb:47:in `load'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/base.rb:66:in `find_asset'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/base.rb:73:in `find_all_linked_assets'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/manifest.rb:142:in `block in find'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/legacy.rb:114:in `block (2 levels) in logical_paths'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/path_utils.rb:225:in `block in stat_tree'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/path_utils.rb:209:in `block in stat_directory'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/path_utils.rb:206:in `each'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/path_utils.rb:206:in `stat_directory'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/path_utils.rb:224:in `stat_tree'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/path_utils.rb:228:in `block in stat_tree'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/path_utils.rb:209:in `block in stat_directory'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/path_utils.rb:206:in `each'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/path_utils.rb:206:in `stat_directory'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/path_utils.rb:224:in `stat_tree'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/path_utils.rb:228:in `block in stat_tree'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/path_utils.rb:209:in `block in stat_directory'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/path_utils.rb:206:in `each'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/path_utils.rb:206:in `stat_directory'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/path_utils.rb:224:in `stat_tree'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/legacy.rb:105:in `each'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/legacy.rb:105:in `block in logical_paths'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/legacy.rb:104:in `each'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/legacy.rb:104:in `logical_paths'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/manifest.rb:140:in `find'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/sprockets/manifest.rb:168:in `compile'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-rails-2.3.3/lib/sprockets/rails/task.rb:70:in `block (3 levels) in define'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-3.5.2/lib/rake/sprocketstask.rb:147:in `with_logger'
/home/vishal/.rvm/gems/ruby-2.2.3/gems/sprockets-rails-2.3.3/lib/sprockets/rails/task.rb:69:in `block (2 levels) in define'
Tasks: TOP => assets:precompile
(See full trace by running task with --trace)
zlib(finalizer): Zlib::GzipWriter object must be closed explicitly.
zlib(finalizer): the stream was freed prematurely.

```

**Comment by vishalzambre** [comment]
will fixing in next PR

**Comment by Mafi88** [comment]
@vishalzambre This issue was already resolved in #6959. Please try with latest master.
